packages/frontEnd/src/views/categoryDetails/filters/FilterByType/FilterByTypeForm.tsx

```diff
@@ -42,14 +48,15 @@ export const FilterByTypeForm = () => {
               id="booking"
               className="form-checkbox rounded"
               type="checkbox"
+              {...register("booking")}
             />
             <label htmlFor="booking" className="ml-2">
               <BookingBadge />
             </label>
           </div>
           <p className="text-cold-gray">({15})</p>
         </div>
-      </form>
+      </div>
```

Comment:
I think you can group checkboxes and return only the one that is currently selected.

Reply:
How?